pkg/roachpb/genbatch/main.go

```diff
@@ -11,11 +11,10 @@
 // This file generates batch_generated.go. It can be run via:
```

Comment:
Since you deleted the `+build gen-batch` tag, this comment is no longer accurate. You should replace it with accurate documentation.